main.go

```diff
@@ -44,6 +44,7 @@ var (
 	stDownloadCmd        = stCmd.Command("download", "Download a ServerTemplate and all associated RightScripts/Attachments to disk")
 	stDownloadNameOrHref = stDownloadCmd.Arg("name|href|id", "Script Name or HREF or Id").Required().String()
 	stDownloadTo         = stDownloadCmd.Arg("path", "Download location").String()
+	stDownloadRsPath     = stDownloadCmd.Flag("rs_path", "Download location for rightscripts and attachments\n\nNote: The rightscript path is always relative to the server template path.\n\nExample: right_st st download --rs_path rightscripts \"My Server Template\"\n\nThis would download the server template yml file in the current directory and create a rightscripts directory in the current directory.").Short('r').String()
```

Comment:
I think this flag should be called simply `script-path` and its description is way too long. Maybe something like:

```go
stDownloadScriptPath = stDownloadCmd.Flag("script-path", "Download RightScripts and their attachments to a subdirectory relative to the download location.").Short('s').String()
```